### ISSUE TITLE
[com_fields] Undefined index: tag

### ISSUE THREAD

**Issue by Quy** [joomla/joomla-cms#14991]
### Steps to reproduce the issue
Go to Contents > Fields.
Create a text field.
In the Fields list, click Check All checkbox.
Click Batch button.
Select from the dropdowns or leave it as it.
Click Process button.

In the PHP error log:
>PHP Notice:  Undefined index: tag in \joomla-cms-staging\libraries\legacy\model\admin.php on line 258

**Comment by laoneo** [comment]
Can you please test pr #14995 if it fixes the issue?

**Comment by Quy** [comment]
Yes. Please fix for Field Groups too. Sorry I should have checked it before.
```
PHP Notice:  Undefined property: FieldsModelGroup::$typeAlias in \joomla-cms-staging\libraries\legacy\model\admin.php on line 222

PHP Notice:  Undefined index: tag in \joomla-cms-staging\libraries\legacy\model\admin.php on line 258
```

**Comment by laoneo** [comment]
I'v changed the pr, so the group should be fixed as well. Guess this issue can be closed as we have a pr.

**Comment by brianteeman** [comment]
Closed as we have a PR for testing #14995
